MetabolicAtlas/MetabolicAtlas
# Fix/1322 make links behave consistently

**Related issue(s) and PR(s)**  
This PR closes #1322

Link targets have been updated to behave consistently as follows:
- Local links open in the current window/tab
- External links open in a new window/tab

**Type of change**  
 
**List of changes made**  
- Add cypress tests of links
- Remove old non-working cypress tests
- Make sure tested vue components work well with cypress expectations
- Make sure all links follow the guideline -> external links `target="_blank"`

**Testing**  
- Check cypress tests
- Run cypress tests `cd frontend && npx cypress run cypress`
- Do some clicking around

**Definition of Done checklist**
